visualizer.py

```diff
@@ -24,16 +24,6 @@
 # Ensure there is only one instance of the script running.
 fh = 0
 
-# make sure connectall.rb file exists and is updated
-if not os.path.exists('/usr/local/bin/connectall.rb'):
-    print("Connectall.rb script not found, creating...")
-    os.mknod('/usr/local/bin/connectall.rb')
-
-if filecmp.cmp('/usr/local/bin/connectall.rb', 'connectall.rb') is not True:
-    print("Connectall.rb script is outdated, updating...")
-    copyfile('connectall.rb', '/usr/local/bin/connectall.rb')
-
```

Comment:
Wouldn't it be wise to keep this part in case you want to update the script in the future? After changing "rb" to "py" of course.

Reply:
I totally agree. Updating an existing visualizer could not be as easy as just hitting update anyways.

Reply:
It might be reasonable.   In my case I don't write these files into /usr/local/bin at all, nor do I want to add any programs there.  The visualizer runs out of a git checkout.

Another possibility would be to just have the visualizer be able to be called instead like:  "visualizer.py --connectall", so that the script in /usr/local/bin is not needed.   And the systemd/udev scripts could be updated to just invoke the visualizer directly for this operation.   Would that be better, or do you want the connectall script to be separated?